components/applications-service/pkg/server/periodic.go

```diff
@@ -201,6 +328,22 @@ func (j *JobRunnerSet) Start(cerealSvc *cereal.Manager) error {
 		return errors.Wrap(err, "failed to register as workflow exector to mark disconnected services")
 	}
 
+	err = cerealSvc.RegisterTaskExecutor(
+		DeleteDisconnectedServicesJobName,
+		j.DeleteDisconnectedServicesExecutor,
+		cereal.TaskExecutorOpts{},
+	)
+
+	if err != nil {
+		return errors.Wrap(err, "failed to register as task exector to delete disconnected services")
```

Comment:
```suggestion
		return errors.Wrap(err, "failed to register as task executor to delete disconnected services")
```
I saw the same typo in at least one other place...